### DISCUSSION TITLE
How to enable CORS Filter in OpenAM?

### DISCUSSION THREAD

**Discussion by yashpatil1998** [OpenIdentityPlatform/OpenAM#575] [Q&A] [answered]
I have tried to refer this [documentation](https://backstage.forgerock.com/docs/am/7/security-guide/enable-cors-support.html#add-cors-config-ui) to enable CORS configuration in the Ui. But the corresponding option isn't available in OpenAM (the same is available in ForgeRock AM). Is there the corresponding available, if not, any configuration steps to enable CORS filter?

I am using [OpenAM 14.7.1](https://hub.docker.com/layers/openidentityplatform/openam/14.7.1/images/sha256-a238a2924f4e2d4c3b7144cce9de83c6f1c916ff930973b333858b3c480f8530?context=explore) image.

**Comment by yashpatil1998** [comment]
Current workaround which I have used is to disable CORS in Safari browser.

**Comment by maximthomas** [comment, marked as answer]
Hi, you need to uncomment the following lines in `WEB-INF/web.xml` file in your OpenAM deployment and configure CORS filter parameters according to your needs

https://github.com/OpenIdentityPlatform/OpenAM/blob/da05a2f102161c5cbbc091ad52f34a3c2cd6ee68/openam-server-only/src/main/webapp/WEB-INF/web.xml#L86-L149

https://github.com/OpenIdentityPlatform/OpenAM/blob/da05a2f102161c5cbbc091ad52f34a3c2cd6ee68/openam-server-only/src/main/webapp/WEB-INF/web.xml#L269-L274
